CMakeLists.txt

```diff
@@ -13,11 +13,20 @@ set(CMAKE_MODULE_PATH
   "${CMAKE_CURRENT_SOURCE_DIR}/cmake/modules"
   )
 
-#Define ROOTbench source tree
+#---Define ROOTbench source tree
 set(ROOTBENCH_SOURCE_DIR ${CMAKE_CURRENT_SOURCE_DIR})
 
+#---Include rootbench options
+include(RootBenchOptions)
+
 include(AddRootBench)
 
+#---ROOTbench dependencies
+find_program(TIME_EXECUTABLE time)
+if(NOT TIME_EXECUTABLE)
+  message(WARNING "/usr/bin/time is requirement for rootbench.git")
```

Comment:
We should probably check if `time -v` works as IIUC the `-v` is a gnu-specific extension.

`time` is a requirement and I think we should report an error instead of a warning.